src/downshift.js

```diff
@@ -947,13 +947,11 @@ class Downshift extends Component {
 
   updateStatus = debounce(() => {
     const state = this.getState()
-    const item = this.items[state.highlightedIndex]
     const resultCount = this.getItemCount()
     const status = this.props.getA11yStatusMessage({
       itemToString: this.props.itemToString,
       previousResultCount: this.previousResultCount,
       resultCount,
-      highlightedItem: item,
```

Comment:
Could we leave this as-is to avoid a breaking change to the API? Also, it's in the typings that you added :)

Reply:
reverted that